### ISSUE TITLE
Typescript definition for method `all` of `LDFeatureStore` is incorrect

### ISSUE THREAD

**Issue by magne4000** [launchdarkly/node-client#77]
#### The definition
https://github.com/launchdarkly/node-client/blob/master/index.d.ts#L271
```typescript
all: (callback: (err: any, res: LDFlagSet) => void) => void;
```

#### The usage
https://github.com/launchdarkly/node-client/blob/master/index.js#L211
```javascript
config.feature_store.all(function(flags) {
  ...
});
```

#### fix
So the TS definition should be something like:
```typescript
all: (callback: (res: LDFlagSet) => void) => void;
```

**Comment by apucacao** [comment]
Thanks for reporting this @magne4000 — we'll get a fix out shortly.

**Comment by apucacao** [comment]
Hi @magne4000 , we just released 3.3.2 which includes your correction. Thanks!
